registrations/models.py

```diff
@@ -1108,7 +1109,22 @@ def send_notification(self, notification_type, access_code=None):
         )
         rendered_body = message[1]
 
-        send_mail(*message, html_message=rendered_body)
+        email = EmailMultiAlternatives(*message)
+        email.attach_alternative(rendered_body, "text/html")  # Optional HTML message
+
+        if notification_type in [
+            SignUpNotificationType.CONFIRMATION,
+            SignUpNotificationType.CONFIRMATION_TO_WAITING_LIST,
+        ]:
+            try:
+                ics_attachment = create_event_ics_file_content(
+                    self.registration.event, self.get_service_language_pk()
+                )
+                email.attach(*ics_attachment, "text/calendar")
+            except ValueError as error:
+                print(str(error))
```

Comment:
```suggestion
                logger.error(error)
```

Reply:
Yeah, I was quite sure that print is not the optimal way to log errors. Thanks for the suggestion. Fixed now